codecept.conf.js

```diff
@@ -41,11 +41,13 @@ const addPages = (include) => {
 
 module.exports.config = {
   tests: `./${TestDir}/**/*_test.js`,
-  timeout: 10000,
+  timeout: 0,
   output: `./${TestDir}/output`,
   helpers: {
     Puppeteer: {
       url: `http://${options.hostname}:${options.port}`,
+      waitForNavigation: "networkidle0",
+      waitForTimeout: 30000,
```

Comment:
These two are what affect I.waitForNavigation() behavior and the default timeouts for I.waitForInvisible() and other waitFor()s...